plugin/output/splunk/splunk.go

```diff
@@ -155,6 +156,7 @@ func (p *Plugin) out(workerData *pipeline.WorkerData, batch *pipeline.Batch) {
 	data.outBuf = outBuf
 
 	for {
+		p.logger.Debugf("Trying to send: %s", string(outBuf))
```

Comment:
Maybe it's better to write before the loop ?

Reply:
Done!

Thanks for noticing